cypress/e2e/stellar-wallet.cy.ts

```diff
@@ -52,3 +52,72 @@ describe('UI Layout', () => {
     });
   });
 });
+
+describe('Dashboard Wallet', () => {
+  describe('Balance information section', () => {
+    it('should show your balance in XLM', () => {
+      cy.get('[data-cy="dashboard-main-container"]').should('exist').and('be.visible');
+      cy.get('[data-cy="dashboard-balance-section-container"]').should('exist').and('be.visible');
+      cy.get('[data-cy="dashboard-balance-information"]').should('exist').and('be.visible');
+      cy.get('[data-cy="dashboard-balance-information"]').find('h3').contains('Your Balance');
+      cy.get('[data-cy="dashboard-balance-information"]').find('p').contains('Lumens (XLM)');
+
+      cy.get('[data-cy="dashboard-balance-btn-container"]').should('exist').and('be.visible');
+      cy.get('[data-cy="dashboard-balance-btn-send"]').should('be.visible').contains('Send');
+      cy.get('[data-cy="dashboard-balance-btn-receive"]').should('be.visible').contains('Receive');
+
+      cy.get('[data-cy="dashboard-balance-public-key-container"]')
+        .should('exist')
+        .and('be.visible');
+      cy.get('[data-cy="dashboard-balance-public-key-container"]')
+        .find('h3')
+        .should('be.visible')
+        .contains('Your Stellar Public Key');
+      cy.get('[data-cy="dashboard-balance-public-key-container"]')
+        .find('code')
+        .invoke('text')
+        .then((text) => {
+          const firstLetter = text.trim()[0];
+          expect(firstLetter).to.equal('G');
+        });
+    });
+  });
+  describe('Payments History', () => {
+    it('should show your payments history', () => {
+      const theadArray = ['DATE/TIME', 'ADDRESS', 'AMOUNT', 'MEMO', 'OPERATION ID'];
```

Comment:
Is not necessary to clarify in this variable name that this is an array, we know that because of the value. We should rename this to a more descriptive variable describing what it stores.